### ISSUE TITLE
Problem for development in Windows 10.

### ISSUE THREAD

**Issue by impossible98** [sharkdp/insect#219]
```
npm WARN rm not removing C:\Users\chris\dev\impossible98\insect\node_modules\.bin\JSONStream.cmd as it wasn't installed by C:\Users\chris\dev\impossible98\insect\node_modules\JSONStream
npm WARN rm not removing C:\Users\chris\dev\impossible98\insect\node_modules\.bin\JSONStream as it wasn't installed by C:\Users\chris\dev\impossible98\insect\node_modules\JSONStreamolution inconsistencies caused by unsynchro                
> psc-package@4.0.0 preinstall C:\Users\chris\dev\impossible98\insect\node_modules\psc-package
> node ./install.js > opn@6.0.0: The package has been renamed to `open`
[2/4] Fetching packages...

> purescript@0.13.6 postinstall C:\Users\chris\dev\impossible98\insect\node_modules\purescriptfrom insta                > install-purescript --purs-ver=0.13.6
[3/4] Linking dependencies...
i Found a cache at C:\Users\chris\AppData\Local\purescript-npm-installer-nodejs\Cache\content-v2\sha512\86\9e
[1/2] ⠂ psc-package
√ Restore the cached 0.13.6 binary for win32 (488ms)
√ Verify the restored binary works correctly (481ms)

Installed to C:\Users\chris\dev\impossible98\insect\node_modules\purescript\purs.bin 60.09 MB


> insect@5.3.0 postinstall C:\Users\chris\dev\impossible98\insect
> psc-package install && npm run browserify && npm run copy

Unable to parse packages.json: Error in $: not enough input
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! insect@5.3.0 postinstall: `psc-package install && npm run browserify && npm run copy`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the insect@5.3.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\chris\AppData\Roaming\npm-cache\_logs\2020-02-04T13_42_07_366Z-debug.log
```

**Comment by sharkdp** [comment]
Thank you very much for the feedback. I don't know what's going on here. It looks like it could be related to a recent change in #217. Maybe @justinwoo could help (if you have time to look at this)?

**Comment by justinwoo** [comment]
No idea what this is really, but let me try checking through some backlogs. @dstcruz would also know a lot more about this than I do

**Comment by justinwoo** [comment]
@impossible98 Could you try the branch in https://github.com/sharkdp/insect/pull/220 sometime? It seems this issue is largely caused by a collision of some gitconfig settings with newlines on Windows and a dependency of psc-package.

**Comment by sharkdp** [comment]
@impossible98 Please let us know if this has been fixed by merging #220.

**Comment by sharkdp** [comment]
Closing this for now.
